uber/site_sections/preregistration.py

```diff
@@ -356,14 +356,6 @@ def register_group_member(self, session, group_id, message='', **params):
             'affiliates': session.affiliates()
         }
 
-    def group_extra_payment_form(self, session, id):
-        attendee = session.attendee(id)
-        cherrypy.session['return_to'] = 'confirm?id={}&message=Extra+payment+undone'.format(attendee.id)
```

Comment:
I'm pretty sure this is a holdover from back before we had plugin-added cost properties. The attendee donation form function handles it a lot better.